index.html

```diff
@@ -16,14 +16,21 @@
 <body>
 <div class="container">
     <h1 class="tittle-brand">Atletismo|<span>Track & Field</span> </h1>
-   
-    <nav class="navbar sticky-top p-0 navbar-expand-sm bg-light navbar-light">
+  
+    <nav class="navbar sticky-top p-0 navbar-expand-lg bg-light navbar-light">
 
         <!-- Brand -->
-        <a class="navbar-brand" href="#">Deporte Rey</a>
-      
+        <a class="navbar-brand" href="#"><span>Deporte Rey</span></a>
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <button class="navbar-toggler custom-focus" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <!--conainer fluid-->
+        <div class="container-fluid">    
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <!-- Links -->
-        <ul class="navbar-nav">
+        <ul class="navbar-nav mr-auto">
           <li class="nav-item">
             <a class="nav-link" href="#">Historia</a>
           </li>
```

Comment:
The same comment about unnecessary white-spacing.

Reply:
fixed